### ISSUE TITLE
Create ajax version of devices page

### ISSUE THREAD

**Issue by neographikal** [yllibed/Zigbee2MqttAssistant#35]
So new devices will pop up without a refresh of the page, helps when adding devices, especially when migrating multiple devices.

**Comment by carldebilly** [comment]
Yep I know.  My goal was to quickly ship something. I have few more features to add before, but that's definitely on the todo list.

**Comment by neographikal** [comment]
No need for excuses, it works great! This would just make it better. Or implement a simple refresh every X seconds button for a quick n dirty solution.

**Comment by carldebilly** [comment]
You're welcome to implement it and do a PR ;-)

**Comment by carldebilly** [comment]
@all-contibutors please add neographikal to ideas

**Comment by carldebilly** [comment]
@all-contributors please add neographikal to ideas

**Comment by vaital3000** [comment]
https://github.com/yllibed/Zigbee2MqttAssistant/pull/261
I tried to do it
